atlas-spec/mpc-engine/src/circuit.rs

```diff
@@ -304,4 +306,42 @@ impl Circuit {
         }
         Ok(output_packed)
     }
+
+    /// Returns the number of gates (i.e. the size) of the circuit.
+    pub fn num_gates(&self) -> usize {
+        self.gates.len()
+    }
+
+    /// Computes the required bucket size for leaky AND triple combination.
+    pub fn and_bucket_size(&self) -> usize {
+        (STATISTICAL_SECURITY as u32 / self.num_gates().ilog2()) as usize
```

Comment:
Generally `into` is preferable when possible because you don't accidentally change values. For constants like `STATISTICAL_SECURITY` it's fine, but for others `into` or `try_into().unwrap()` surface errors.